packages/web-components/fast-foundation/src/badge/badge.ts

```diff
@@ -17,7 +17,7 @@ export class Badge extends FoundationElement {
     public fill: string;
 
     /**
-     * Indicates the badge should have a filled style.
+     * Indicates the badge's color.
      *
      * @public
```

Comment:
is this private?
```suggestion
     * @private
```

Reply:
I don't think so, I think it should remain public. Is there a reason you're changing it?

Reply:
The reason is it doesn't appear in the [readme's API](https://github.com/microsoft/fast/blob/b816a09d0cf393567396e3f5a8e06312798dc05e/packages/web-components/fast-foundation/src/badge/badge.spec.md#api)

Reply:
I think we should leave this public and then just back-port this to the spec.